SpatialGDK/Source/SpatialGDK/Public/Utils/ComponentFactory.h

```diff
@@ -48,8 +48,8 @@ class SPATIALGDK_API ComponentFactory
 
 	Worker_ComponentData CreateInterestComponentData(UObject* Object, const FClassInfo& Info);
 	Worker_ComponentUpdate CreateInterestComponentUpdate(UObject* Object, const FClassInfo& Info);
-	improbable::Interest CreateInterestComponent(UObject* Object, const FClassInfo& Info);
-	void AddObjectToComponentInterest(UObject* Object, UObjectPropertyBase* Property, uint8* Data, improbable::ComponentInterest& ComponentInterest);
+	SpatialGDK::Interest CreateInterestComponent(UObject* Object, const FClassInfo& Info);
```

Comment:
```suggestion
	Interest CreateInterestComponent(UObject* Object, const FClassInfo& Info);
```
Unnecessary since we're in the namespace.

Reply:
Looks like your PR addresses this.